pom.xml

```diff
@@ -141,67 +160,11 @@
   </dependencies>
 
   <profiles>
-    <profile>
-      <id>indy</id>
-      <dependencies>
-        <dependency>
-          <groupId>${groovyGroupId}</groupId>
-          <artifactId>groovy</artifactId>
-          <version>${groovyVersion}</version>
-          <classifier>indy</classifier>
-          <scope>test</scope>
-        </dependency>
-        <dependency>
-          <groupId>${groovyGroupId}</groupId>
-          <artifactId>groovy-ant</artifactId>
-          <version>${groovyVersion}</version>
-          <classifier>indy</classifier>
-          <scope>test</scope>
-        </dependency>
-        <dependency>
-          <groupId>${groovyGroupId}</groupId>
-          <artifactId>groovy-groovydoc</artifactId>
-          <version>${groovyVersion}</version>
-          <classifier>indy</classifier>
-          <scope>test</scope>
-        </dependency>
-      </dependencies>
-      <build>
-        <plugins>
-          <plugin>
-            <groupId>org.apache.maven.plugins</groupId>
-            <artifactId>maven-invoker-plugin</artifactId>
-            <configuration>
-              <profiles>
-                <profile>indy</profile>
-              </profiles>
-            </configuration>
-          </plugin>
-        </plugins>
-      </build>
-    </profile>
     <profile>
       <id>nonindy</id>
```

Comment:
Since you removed the nonindy profiles from the integration tests, I think you don't need this profile here anymore.

Reply:
Thanks, updated.